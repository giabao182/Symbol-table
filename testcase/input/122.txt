INSERT a number
BEGIN
INSERT b number
BEGIN
INSERT c string
PRINT
END
RPRINT
END