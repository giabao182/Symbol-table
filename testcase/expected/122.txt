success
success
success
a//0 b//1 c//2
b//1 a//0